### PR TITLE
Switch to PROD pool

### PR DESCRIPTION
**Pull request by NikolaMilosavljevic**

### CODE REVIEW

File: .vsts.pipelines/builds/ci-internal.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 jobs:
 - template: matrix.yml
   parameters:
-    windowsPoolName: NetCoreInternal-Int-Pool
+    windowsPoolName: NetCoreInternal-Pool
```
Comment (adaggarwal) [inline .vsts.pipelines/builds/ci-internal.yml]: This might only affect windows pool 
cc @crummel

**Comment by crummel** [comment]
There's one more reference here, I think: https://github.com/dotnet/source-build/blob/6df8641e6f585bc6566c0ded29b6773a3a726f3f/.vsts.pipelines/builds/matrix.yml#L9.

**Comment by NikolaMilosavljevic** [comment]
> There's one more reference here, I think: 
>   
>     
>       source-build/.vsts.pipelines/builds/matrix.yml
>     
>     
>          Line 9
>       in
>       6df8641
>     
>     
>     
>     
> 
>         
>           
>            windowsPoolName: NetCorePublic-Int-Pool 
>         
>     
>   
> 
> .

I've updated the PR.

**Comment by crummel** [comment]
/azp run
(I don't think this is a bad idea even if it wasn't the immediate problem)

**Pull request merged**